src/adagrams.js

```diff
@@ -1,6 +1,83 @@
 const Adagrams = {
   drawLetters() {
-    // Implement this method for wave 1
+    let letterPool = 'AAAAAAAAABBCCDDDDEEEEEEEEEEEEFFGGGHHIIIIIIIIIJKLLLLMMNNNNNNOOOOOOOOPPQRRRRRRSSSSTTTTTTUUUUVVWWXYYZ'.split('');
+    let hand = [];
+    for (let i = 0; i < 10; i += 1) {
+      const letterIndex = letterPool.indexOf(letterPool[Math.floor(Math.random()*letterPool.length)]);
+      hand.push(letterPool[letterIndex]);
+      letterPool.splice(letterIndex, 1);
+    }
+    return hand;
+  },
+
+  usesAvailableLetters(input, lettersInHand) {
+    const inputArr = input.split('');
+    for (const char in inputArr) {
+      if (!lettersInHand.includes(inputArr[char])) {
+        return false;
+      }
+      const ltrIndex = lettersInHand.indexOf(inputArr[char]);
+      lettersInHand.splice(ltrIndex, 1);
+    }
+    return true;
+  },
+
+  scoreWord(word) {
+    let wordCap = word.toUpperCase();
+    let score = 0;
+    for (const letter in wordCap) {
+      if (['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T'].includes(wordCap[letter])) {
```

Comment:
Nice work with this if statement. What would this code have looked like with a hash?